tensorflow/python/ops/nn_ops.py

```diff
@@ -3363,6 +3363,61 @@ def sparse_softmax_cross_entropy_with_logits(
       else:
         return cost
 
+@tf_export("nn.sparse_softmax_cross_entropy_with_logits", v1=[])
+def sparse_softmax_cross_entropy_with_logits_v2(
+    labels=None,
```

Comment:
The `None` defaults for `labels` and `logits` should be removed, because these two arguments are always required.
They are needed in v1, however, because v1 uses sentinels.

Reply:
@ppwwyyxx Thanks. The PR has been updated.